.devops/deploy-pipelines.yml

```diff
@@ -85,15 +106,15 @@ stages:
               command: 'Build'
               Dockerfile: '$(Build.Repository.LocalPath)/$(dockerfileRelativePath)'
               buildContext: '$(Build.Repository.LocalPath)'
-              arguments: --build-arg APP_IMAGE=$(DOCKER_BASE_IMAGE_NAME):${{ parameters.version }}
+              arguments: --build-arg APP_IMAGE=$(DOCKER_BASE_IMAGE_NAME):$(dockerImageVersion)
               tags: |
                 $(Build.BuildId)
```

Comment:
can we add the branch name to be more simple to understand the buildId?

Reply:
the branch name in this case will be always "main"

Reply:
we could user buildNumber? (eg: 20220310.14, or the date plus the number of execution at the date)

Reply:
done